RELEASES.md

```diff
@@ -0,0 +1,90 @@
+A major release series is comprised of all minor and patch releases made under the same major version number. 
+The series continues to receive bug fixes (released as minor or patch releases) until it reaches end of life. 
+The date when a major release series reaches end of life is determined by one of the following methods:
+
+- If there is no known chain using a major release series, then it reached end of life. 
+  This is a temporary policy that is possible due to the relatively low number of consumer chains.
+- If the next major release is made within the first 6 months, then the end of 
+  life date of the major release series is 1 year after its initial release.
+- If the next major release is made 6 months after the initial release, then the 
+  end of life date of the major release series is 6 months after the release date 
+  of the next major release.
+
+Only the following major release series have a stable release status. 
+All missing minor release versions have been discontinued.
+
+| Release | End of Life Date |
+|---------|------------------|
+| `v1.2.x` | February 21, 2024 |
+| `v2.0.x` | June 09, 2024 |
+| `v2.1.x-provider-lsm` | June 09, 2024 |
+| `v2.3.x-provider-lsm` | June 09, 2024 |
+| `v3.1.x` | July 10, 2024 |
+| `v3.2.x` | July 10, 2024 |
+
+**Note**: As of [Gaia v12.0.0](https://github.com/cosmos/gaia/releases/tag/v12.0.0), 
+the Cosmos Hub uses a fork of Cosmos SDK ([v0.45.16-ics-lsm](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.16-ics-lsm)) 
+that contains the Liquid Staking Module (LSM). 
+This means the Cosmos Hub requires a fork of ICS. 
+This fork is maintained by the development team and released using the `-lsm` prefix. 
+As soon as the Cosmos Hub uses mainline Cosmos SDK, the `-lsm` releases will reach end of life. 
+
+## Version Matrix
+
+Versions of Golang, IBC, Cosmos SDK and CometBFT used by ICS in the currently active releases:
+
+| ICS | Golang | IBC | Cosmos SDK | CometBFT | Note |
+|-----|--------|-----|------------|----------|------|
+| [v1.2.0-multiden](https://github.com/cosmos/interchain-security/releases/tag/v1.2.0-multiden) | 1.18 | v4.2.0 | v0.45.15-ics | v0.34.27 | Consumer only |
+| [v2.0.0](https://github.com/cosmos/interchain-security/releases/tag/v2.0.0) | 1.19 | v4.4.2 | v0.45.15-ics | v0.34.28 | 
+| [v2.1.0-provider-lsm](https://github.com/cosmos/interchain-security/releases/tag/v2.1.0-provider-lsm) | 1.19 | v4.4.2 | v0.45.16-ics-lsm | v0.34.28 | Provider only (Cosmos Hub specific) |
+| [v2.3.0-provider-lsm](https://github.com/cosmos/interchain-security/releases/tag/v2.3.0-provider-lsm) | 1.19 | v4.4.2 | v0.45.16-ics-lsm | v0.34.28 | Provider only (Cosmos Hub specific) |
+| [v3.1.0](https://github.com/cosmos/interchain-security/releases/tag/v3.1.0) | 1.20 | v7.1.0 | v0.47.3 | v0.37.2 |
+
+### Backwards Compatibility
+
+A MAJOR version of ICS will always be backwards compatible with the previous MAJOR version of ICS. 
```

Comment:
I do not fully undertand this. The way I read this, it seems that we had a _MAJOR_ version bump without having a library API breaking change and hence we're still backwards compatible. Is this the case?

Reply:
Backwards compatible doesn't mean that we don't have library API breaking changes. It means that a consumer running the new version will work with a provider running the previous version (and the other way around).